.gitignore

```diff
@@ -108,6 +108,9 @@ ENV/
 # snapshots
 *.tar
 
+# pylint config is left at dev's discretion, CI uses ruff/isort/black instead
+.pylintrc
```

Comment:
Is it really need? If run locally pylint without disable tags, pylint will show many errors.

Reply:
I have pylint integration in my IDE and it respects my local .pylintrc for highlighting only the stuff I find to be helpful - this is the use case I'm enabling. A global pylint run will be broken indeed.